src/VkBootstrap.h

```diff
@@ -802,7 +880,16 @@ struct Swapchain {
     // structure.
     Result<std::vector<VkImageView>> get_image_views();
     Result<std::vector<VkImageView>> get_image_views(const void* pNext);
-    void destroy_image_views(std::vector<VkImageView> const& image_views);
+    void destroy_image_views(size_t count, VkImageView const* image_views);
+    void destroy_image_views(std::vector<VkImageView> const& image_views) {
+        destroy_image_views(image_views.size(), image_views.data());
+    }
+
+#if VKB_SPAN_OVERLOADS
+    void destroy_image_views(std::span<const VkImageView> image_views) {
+        destroy_image_views(image_views.size(), image_views.data());
+    }
```

Comment:
I'm not seeing the point of this as much, since vk-bootstrap only returns the images in a vector. If tehre was a way to get image views using your own storage then these overloads make more sense.

Reply:
You could extract the images into your own storage, and limiting that is a bit pointless imo.

Reply:
Very true, so I'll resolve this since its fine as it is (barring moving implementations to .cpp file where appropriate).